spec/models/vm_or_template_spec.rb

```diff
@@ -658,35 +658,49 @@
     end
   end
 
-  context "#supports_terminate?" do
-    let(:ems_does_vm_destroy) { FactoryBot.create(:ems_vmware) }
-    let(:ems_doesnot_vm_destroy) { FactoryBot.create(:ems_storage) }
+  context "#supports_control?" do
```

Comment:
Most of these were actually testing `supports_control?`

Reply:
Was `supports_control?` supposed to stick around?

https://github.com/ManageIQ/manageiq/blob/master/app/models/mixins/supports_feature_mixin.rb#L66-L67

Reply:
Well it is still an internal helper method, e.g.: https://github.com/ManageIQ/manageiq-providers-amazon/pull/626/files#diff-8b4749bdfb860692ef6e4651bf0faa5aR9